### PR TITLE
logging integrations: add SwiftyBeaver and CocoaLumberjack integrations

### PR DESCRIPTION
**Pull request by Augustyniak**
- Add the implementation of `CaptureSwiftyBeaverIntegration` and `CaptureCocoaLumberjackIntegration` libraries. Even though this PR adds the implementation of the libraries they are not being released as part of SwiftPM or CocoaPods distribution yet - this is going to be done in follow up PRs.
- Now that we have Swift code in the repo add various linters to enforce code style consistency: `swiftlint`, `SwiftFormat` and `DrString`.
- expand existing repo README

### CODE REVIEW

File: Sources/CaptureCocoaLumberjackIntegration/CaptureDDLogger.swift [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+        case .warning:
+            self = .warning
+        case .info:
+            self = .info
+        case .debug:
+            self = .debug
+        case .verbose, .all:
+            self = .trace
+        default:
+            self = .trace
```
Comment (murki) [inline Sources/CaptureCocoaLumberjackIntegration/CaptureDDLogger.swift:55]: not super important, and we never really discussed it, but for whatever reason I set the default to be debug on android timber

**Review by Reflejo — APPROVED** [review]

File: Sources/CaptureCocoaLumberjackIntegration/CaptureDDLogger.swift [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    var logFormatter: DDLogFormatter?
+
+    func log(message logMessage: DDLogMessage) {
+        guard let level = LogLevel(logMessage.level) else {
+            return
+        }
+
+        Capture.Logger.log(
+            level: level,
+            message: logMessage.message
```
Comment (murki) [inline Sources/CaptureCocoaLumberjackIntegration/CaptureDDLogger.swift:20]: fyi on Android I decided to attach an extra filed `_source` with some name that identifies the library that is sending the logs ("Timber" in my case)

File: Sources/CaptureSwiftyBeaverIntegration/CaptureSwiftyBeaverLogger.swift [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+/// Capture SDK.
+final class CaptureSwiftyBeaverLogger: BaseDestination {
+    override func send(
+        _ level: SwiftyBeaver.Level,
+        msg: String,
+        thread: String,
+        file: String,
+        function: String,
+        line: Int,
+        context: Any? = nil
```
Comment (murki) [inline Sources/CaptureSwiftyBeaverIntegration/CaptureSwiftyBeaverLogger.swift:15]: shouldn't we also fwd the rest of these ^ fields to our logger?

File: Sources/CaptureCocoaLumberjackIntegration/CaptureDDLogger.swift (hunk 2 above)
Comment (murki) [reply Sources/CaptureCocoaLumberjackIntegration/CaptureDDLogger.swift:20]: how are they not internal? In my view "internal" means anything added automatically by code that we provide, which is the case here, wouldn't you agree?

File: Sources/CaptureSwiftyBeaverIntegration/CaptureSwiftyBeaverLogger.swift (hunk 3 above)
Comment (murki) [reply Sources/CaptureSwiftyBeaverIntegration/CaptureSwiftyBeaverLogger.swift:15]: what about thread?

**Review by murki — APPROVED** [review]

File: Sources/CaptureSwiftyBeaverIntegration/CaptureSwiftyBeaverLogger.swift (hunk 3 above)
Comment (murki) [reply Sources/CaptureSwiftyBeaverIntegration/CaptureSwiftyBeaverLogger.swift:15]: For threads we could just add it as one of the fields no? I don't see the value on having to wait until we implement our own approach if they already provide it

**Pull request merged**